### DISCUSSION TITLE
Evolução da Robótica

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#476] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/evolucao-da-robotica/

**Comment by maryon-telvanni** [comment]
Isso é tão fantástico quanto aterrorizante :flushed:

**Comment by angellisy** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/videos/evolucao-da-robotica/#comment_1142140113>):

> Isso é tão fantástico quanto aterrorizante 😳

Né?

**Comment by brunoaugusto** [comment]
![null](<https://i.imgur.com/t0FiJwF.png>)
